ext/curl/interface.c

```diff
@@ -3468,6 +3469,7 @@ static void curl_free_obj(zend_object *object)
 	zval_ptr_dtor(&ch->handlers.std_err);
 	if (ch->header.str) {
 		zend_string_release_ex(ch->header.str, 0);
+		ch->header.str = NULL;
```

Comment:
This is unlikely to be necessary, as this function destroys `ch` and it cannot be used afterwards anymore.

Reply:
I'm not sure about the usefulness of this line. You, as a developer, know better whether need it here. My code works fine with it.